picard/pluginmanager.py

```diff
@@ -342,13 +369,37 @@ def _load_plugin(self, name):
                           plugin.version,
                           ", ".join([v.short_str() for v in
                                      sorted(compatible_versions)]))
+                log.info(f"[pluginmanager] Successfully loaded plugin: {plugin.name} (version {plugin.version})")
                 plugin.compatible = True
                 setattr(picard.plugins, name, plugin_module)
                 if existing_plugin:
                     self.plugins[existing_plugin_index] = plugin
                 else:
                     self.plugins.append(plugin)
+                # --- BEGIN: Workaround: Hook-Registrierungsfunktionen explizit aufrufen ---
+                # Versuche, alle bekannten Hook-Registrierungsfunktionen aufzurufen
+                tagger_instance = None
+                try:
+                    import picard.tagger
+                    tagger_instance = picard.tagger.Tagger.instance() if hasattr(picard.tagger.Tagger, 'instance') else None
+                except Exception as e:
+                    log.error(f"[pluginmanager] Fehler beim Zugriff auf Tagger-Instanz: {e}")
+                for hook_func in [
+                    'register_file_post_load_processor',
+                    'register_album_metadata_processor',
+                    'register_file_post_save_processor',
+                    'register_track_metadata_processor',
+                    'register_album_post_removal_processor',
+                ]:
+                    if hasattr(plugin_module, hook_func):
+                        try:
+                            getattr(plugin_module, hook_func)(tagger_instance)
+                            log.info(f"[pluginmanager] Workaround: {hook_func} im Plugin {name} explizit aufgerufen.")
+                        except Exception as e:
+                            log.error(f"[pluginmanager] Fehler beim expliziten Aufruf von {hook_func} in Plugin {name}: {e}")
+                # --- END: Workaround ---
```

Comment:
More comments and log messages in German, more log.info messages that should be log.debug, and more lists that should be global or class constants. And it doesn't seem like a workaround either - but I am not sure what it is supposed to do or indeed whether hasattr actually does what you think it is going to do.